saleor/order/models.py

```diff
@@ -287,7 +288,7 @@ class Order(ModelWithMetadata):
     )
     redirect_url = models.URLField(blank=True, null=True)
     search_document = models.TextField(blank=True, default="")
-
+    search_vector = SearchVectorField(blank=True, null=True)
```

Comment:
Should we remove `search_document` because we remove the function responsible for updating it?

Reply:
I don't want to break the database during upgrade, it was also done like that for products.